SDWebImage/SDWebImageDownloaderOperation.h

```diff
@@ -82,7 +82,7 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadFinishNotification
  * @note This value may enhance the performance if you don't want progress callback too frequently.
  * Defaults to 0, which means each time we receive the new data from URLSession, we callback the progressBlock immediately.
  */
-@property (assign, nonatomic) NSTimeInterval minimumProgressInterval;
+@property (assign, nonatomic) double minimumProgressInterval;
```

Comment:
`NSTimeInterval` is typedef of `double`, but let's replace to `double`. Because its semantic maybe not we want. 

> NSTimeInterval is always specified in seconds; it yields sub-millisecond precision over a range of 10,000 years.

Reply:
Yeap, we need to consistently change all the `minimumProgressInterval`, from `NSTimeInterval` to `double`.